### ISSUE TITLE
Unhandled Rejection at: Promise, unsupported protocol HTTP

### ISSUE THREAD

**Issue by Bwen** [Mockgoose/Mockgoose#64]
https://imgur.com/a/DNbEu

I'm running into this issue and i'm not 100% what I need to do to fix it... 
docker node:8.9-alpine
mockgoose v7.3.3
feathersjs v3.0.0

Happens when calling mockgoose.prepareStorage(), promises never finished.

Thanks for the help.

**Comment by nedyalkov** [comment]
We are having the same problem with our tests. The issue seems to be caused by this commit: https://github.com/winfinit/mongodb-download/commit/637ed2f9e1a6d6a8bb4a41509608545765015c4e (Mockgoose depends on mongodb-prebuilt and it depends on mongodb-download).

**Comment by weeksling** [comment]
I am getting the very same issue. I tried replacing the http in DOWNLOAD_URI with https, and the code works again.

Unsure how to patch this in my package.json until this issue is fixed

**Comment by voodee** [comment]
+1

**Comment by m-ssilva** [comment]
Same here

**Comment by hcnode** [comment]
This happened in my travis unit test, finally I add `npm i mongodb-download@2.2.3` in "before_install" script in .travis.yml, and it work.

**Comment by franky-continu** [comment]
+1

```
(node:23) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: Protocol "http:" not supported. Expected "https:"
```

It just started to happen all of a sudden

**Comment by vasilis-exo** [comment]
+1

**Comment by flx-l** [comment]
There is a pending pull request (https://github.com/winfinit/mongodb-download/pull/27), that fixes this error.

**Comment by vladotesanovic** [comment]
I created temporary npm package with updated @flx-l mongodb-prebubuilt package.

```
npm install mockgoose-fix --save-dev
```

I m not able to wait for them to update, as i need this in my system

**Comment by piyushbeli** [comment]
@vladotesanovic Thanks. Works perfectly.

**Comment by ellenaua** [comment]
Installation of mongodb-download 2.2.3 also helps

`npm install mongodb-download@2.2.3`

**Comment by ericadamski** [comment]
I cannot seem to get my mongo mocked, still running into this issue. What is the progress here? Anyway I can help?

**Comment by victorperin** [comment]
i fixed using `mockgoose-fix`. Are you importing it in place of `mockgoose` default?

**Comment by ericadamski** [comment]
@victorperin that's good info, I was not.

**Comment by sjorssnoeren** [comment]
@vladotesanovic Thanks for the effort of creating this! I've tried it and locally it works just fine, but when running on CI, I receive: "unable to get signature content" with a 403 statusCode. Any idea what may cause this?

**Comment by vladotesanovic** [comment]
@sjorssnoeren how you run on CI ```npm run test``` ```npm run tsc``` ... ?

**Comment by sjorssnoeren** [comment]
@vladotesanovic I use npm test. My package.json test command includes: "./node_modules/eslint/bin/eslint.js lib test --fix && ./node_modules/nyc/bin/nyc.js ./node_modules/mocha/bin/mocha test --recursive"

**Comment by vladotesanovic** [comment]
@sjorssnoeren https://travis-ci.org/vladotesanovic/Mockgoose

**Comment by sjorssnoeren** [comment]
@vladotesanovic I receive the following output on CircleCI: https://gist.github.com/sjorssnoeren/28302c5cd9712bc1a04179f1bfe47f0f - Thanks for your help so far!

**Comment by vladotesanovic** [comment]
@sjorssnoeren 
set version to:
```
mockgoose.helper.setDbVersion('3.4.3');
```

**Comment by binary1230** [comment]
I posted a workaround in #67 that works for me

**Comment by LoneRifle** [comment]
Fixed by winfinit/mongodb-download#27
